Da regulatory formatting editing

anselld opened the pull request:
Hansie and Kev, please could you have a look and let me know if there are any issues. I am having a problem with escaping the dollar symbol in footnote 17. Please let me know what I'm doing wrong. Thanks.
Also, what is Louis's address? I need to insert it under 'Contributors'.

anselld commented:
Thanks, Hansie - the footnote 17 problem has been resolved. I just need Louis's address for the 'Contributors' section.

hansieodendaal commented:
Hi @anselld and @lwillacy. 

1. The formatting and conversion into an online document looks good. 

1. The concept of a footnote, which works well in Word, does not work that well on a web page. Suggest to minimize use of footnotes and insert its text into the main text where possible. 

1. We have two occurrences of a footnote linked to a reference, and another where two footnotes follow in series. Suggest to simplify that as well.

    ![image](https://user-images.githubusercontent.com/39146854/55785204-4d5b0500-5ab2-11e9-95f9-052f1c21cf3c.png)

    ![image](https://user-images.githubusercontent.com/39146854/55785213-521fb900-5ab2-11e9-9177-de34242f7866.png)

    ![image](https://user-images.githubusercontent.com/39146854/55785227-577d0380-5ab2-11e9-8e59-5f8997025ccd.png)

Regards,
Hansie.

anselld commented:
Hi @lwillacy and @hansieodendaal.
1. @hansieodendaal, thanks for your comments about the footnotes. I agree that they can make legibility more difficult for the reader. Louis, if you agree, I can update the report in line with Hansie's comments.
2. @lwillacy , please have a look at my editing and formatting changes. Let me know if you'd like any changes and I'll update the report accordingly.
Regards
Denise

hansieodendaal commented:
Hi @anselld I think you can go ahead to:

1. work the footnotes into the text where it makes senses and if not referenced more than once;

1. conclude your editing for @lwillacy to review.

Regards

anselld commented:
Hi @lwillacy, @hansieodendaal and @Kevoulee 
Thanks very much for your comments. I'll make the changes suggested and send them for review.

anselld commented:
Hi @lwillacy and @hansieodendaal 
I've moved the footnote text into the main body of the report where possible, and updated the list of footnotes.
@lwillacy , regarding the text references to the four orders (Munchee, Airfox, Paragon and Gladius), may I insert a link after each occurrence to [1, Section 8A]? I will then update the references in the bulleted list in the introduction to also read  [1, Section 8A].
Please let me have your comments - I will make the changes.
Thanks!

lwillacy commented on src/digital-assets/howey-application-to-blockchain/MainReport.md:
```diff
@@ -144,63 +168,110 @@ participants seek a greater understanding of whether or not a digital asset is a
+> Paragon also maintained… a 'Controlled Reserve Fund' to keep the price of PRG 'stable'. If the price of PRG tokens were 
+> to drop significantly, the Controlled Reserve Fund would 'intervene by buying back PRG in an effort to stabilize the 
+> market price'. Conversely, the Controlled Reserve Fund would '[r]elease PRG to the markets if PRG deflates too fast 
+> and pushes token prices up too rapidly'"); Guillermo Jimenez, SEC's Crypto Czar: Stablecoins might be violating 
```
This reference is not part of the Paragon Order.  I suggest a separate intro:

"Further, the SEC's Senior Advisor for Digital Assets, Valerie Szczepanik, noted promoter efforts in this area may implicate U.S. securities law recently stating she has "seen stablecoins that purport to control price through some kind of pricing mechanism… controlled through supply and demand in some way to keep the price within a certain band", and "[W]here there is one central party controlling the price fluctuation over time, [that] might be getting into the land of securities."

CITE: Guillermo Jimenez, SEC's Crypto Czar: Stablecoins might be violating 
> securities laws Decrypt (2019), https://decryptmedia.com/5940/secs-crypto-czar-stablecoins-might-be-violating-securities-laws (accessed March 19, 2019)

anselld replied:
Thanks, Louis - I've made the change.

lwillacy commented on src/digital-assets/howey-application-to-blockchain/MainReport.md:
```diff
@@ -115,17 +127,29 @@ exists mean that any expectation of profits token purchasers may have does not c
-a security", and posed a set of six questions<sup>[f8][footnote_8~]</sup> <sup>[f9][footnote_9~]</sup> directly related 
-to the application of the "efforts of others" prong of the *Howey* test to offerings and sales of digital assets. He 
-noted two cases where he believed this was indeed possible:
+a security", and posed a set of six questions directly related to the application of the "efforts of others" prong of the *Howey* test to offerings and sales of digital assets.
```
Bringing the questions into the text has broken up the flow here.  I suggest rewriting as follows:

"Background
On 14 June 2018, Hinman delivered a speechf2 addressing whether "a digital asset that was originally offered in a securities offering [can] ever later be sold in a manner that does not constitute an offering of a security", and noted two cases where he believed this was indeed possible:

"where there is no longer any central enterprise being invested in", e.g. purchases of the digital assets related to a decentralized enterprise or network; and
"where the digital asset is sold only to be used to purchase a good or service available through the network on which it was created", e.g. purchases of digital assets for a consumptive purpose.

He posed a set of six questions directly related to the application of the "efforts of others" prong of the Howey test to offerings and sales of digital assets.

The questions were:

Is there a person or group that has sponsored or promoted the creation and sale of the digital asset, the efforts of whom play a significant role in the development and maintenance of the asset and its potential increase in value?
Has this person or group retained a stake or other interest in the digital asset such that it would be motivated to expend efforts to cause an increase in value in the digital asset? Would purchasers reasonably believe such efforts will be undertaken and may result in a return on their investment in the digital asset?
Has the promoter raised an amount of funds in excess of what may be needed to establish a functional network, and, if so, has it indicated how those funds may be used to support the value of the tokens or to increase the value of the enterprise? Does the promoter continue to expend funds from proceeds or operations to enhance the functionality and/or value of the system within which the tokens operate?
Are purchasers 'investing', that is seeking a return? In that regard, is the instrument marketed and sold to the general public instead of to potential users of the network for a price that reasonably correlates with the market value of the good or service in the network?
Does application of the Securities Act protections make sense? Is there a person or entity others are relying on that plays a key role in the profit-making of the enterprise such that disclosure of their activities and plans would be important to investors? Do informational asymmetries exist between the promoters and potential purchasers/investors in the digital asset?
Do persons or entities other than the promoter exercise governance rights or meaningful influence?

(He also posed a separate set of seven questions exploring "contractual or technical ways to structure digital assets so they function more like a consumer item and less like a security".)"

anselld replied:
Thanks, changes made accordingly.

lwillacy commented on src/digital-assets/howey-application-to-blockchain/MainReport.md:
```diff
@@ -115,17 +127,29 @@ exists mean that any expectation of profits token purchasers may have does not c
+
+He also posed a separate set of seven questions exploring "contractual or technical ways to structure digital assets so they function more like a consumer item and less like a security".
+
+Hinman noted two cases where he believed this was indeed possible:
```
In line 141, please add an 's' to "expectation".  IE "purpose and priming purchasers' expectations,"

anselld replied:
Corrected, thanks.

lwillacy commented:
Yes - text references to the orders may be followed by links.  Each of the
orders would be listed in [1, Section 8A], correct?

On Wed, Apr 10, 2019 at 9:35 AM Denise Ansell <notifications@github.com>
wrote:

> Hi @lwillacy <https://github.com/lwillacy> and @hansieodendaal
> <https://github.com/hansieodendaal>
> I've moved the footnote text into the main body of the report where
> possible, and updated the list of footnotes.
> @lwillacy <https://github.com/lwillacy> , regarding the text references
> to the four orders (Munchee, Airfox, Paragon and Gladius), may I insert a
> link after each occurrence to [1, Section 8A]? I will then update the
> references in the bulleted list in the introduction to also read [1,
> Section 8A].
> Please let me have your comments - I will make the changes.
> Thanks!
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/tari-labs/tari-university/pull/141#issuecomment-481766230>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AL9F7FSVObN0XC4AFt7GtMY8TdBACbpdks5vfhLUgaJpZM4ch0-U>
> .
>

anselld commented on src/digital-assets/howey-application-to-blockchain/MainReport.md:
```diff
@@ -144,63 +168,110 @@ participants seek a greater understanding of whether or not a digital asset is a
+identifiable "person or group" that continues to play a significant role, especially as compared to the role that the 
+dispersed stakeholders play [[2], Section 4]<sup>[f3][footnote_3~]</sup>. 
+
+The SEC has placed particular attention on promoter efforts to impact a token's supply and/or demand. It has also 
```
Agreed.

anselld commented:
Hi @hansieodendaal, I've made the changes requested by Louis. I have pushed and commited the latest changes. Please could you let me know why the link in [6] under 'Decentralization' is not rendering correctly. Thank you.

hansieodendaal commented on src/digital-assets/howey-application-to-blockchain/MainReport.md:
```diff
@@ -0,0 +1,372 @@
+[5]: https://www.sec.gov/Article/whatwedo.html
+"U.S. Securities and Exchange Commission, What We Do"
+
+[[6]] Guillermo Jimenez, SEC's Crypto Czar: "Stablecoins might be violating securities laws", Decrypt (2019) [online]. Available: <https://decryptmedia.com/5940/secs-crypto-czar-stablecoins-might-be-violating-securities-laws>. Date accessed: 2019-03-19.
```
```suggestion
[[6]] Guillermo Jimenez, SEC's Crypto Czar: "Stablecoins might be violating securities laws", Decrypt (2019) [online]. 
Available: https://decryptmedia.com/5940/secs-crypto-czar-stablecoins-might-be-violating-securities-laws. Date accessed: 2019-03-19.
```

hansieodendaal commented on src/digital-assets/howey-application-to-blockchain/MainReport.md:
```diff
@@ -0,0 +1,372 @@
+
+[[6]] Guillermo Jimenez, SEC's Crypto Czar: "Stablecoins might be violating securities laws", Decrypt (2019) [online]. Available: <https://decryptmedia.com/5940/secs-crypto-czar-stablecoins-might-be-violating-securities-laws>. Date accessed: 2019-03-19.
+
+[6]: https://decryptmedia.com/5940/secs-crypto-czar-stablecoins-might-be-violating-securities-lawsquot;Stablecoins might be violating securities laws
```
```suggestion
[6]: https://decryptmedia.com/5940/secs-crypto-czar-stablecoins-might-be-violating-securities-lawsquot
```

hansieodendaal commented on src/digital-assets/howey-application-to-blockchain/MainReport.md:
```diff
@@ -0,0 +1,372 @@
+[[6]] Guillermo Jimenez, SEC's Crypto Czar: "Stablecoins might be violating securities laws", Decrypt (2019) [online]. Available: <https://decryptmedia.com/5940/secs-crypto-czar-stablecoins-might-be-violating-securities-laws>. Date accessed: 2019-03-19.
+
+[6]: https://decryptmedia.com/5940/secs-crypto-czar-stablecoins-might-be-violating-securities-lawsquot;Stablecoins might be violating securities laws
+
```
```suggestion
```

anselld replied:
Thanks, Hansie.

anselld commented:
Hi @hansieodendaal, thanks for your help with the reference. Please let me have your comments.
Hi @lwillacy , I have made the changes you requested, and have pushed and committed the latest changes.  Please let me know it you would like any other changes.

hansieodendaal commented:
Hi @lwillacy, to view the rendered version of your report on the current branch please do the following (and forgiveness asked for the detailed instructions if you know all/some of this already):

1. Install prerequisites:

    1. Install rust on your system (see https://www.rust-lang.org/tools/install).

    2. Install mdbook on your system (see https://github.com/rust-lang-nursery/mdBook), use option 2.

    3. Install git on your system (see https://www.atlassian.com/git/tutorials/install-git)

2. Download and sync the TLU repository to your local machine:

    1. Open a command console (terminal window) in your home folder.

    2. Execute `git clone https://github.com/tari-labs/tari-university.git` to sync to the master branch of the repository. A new folder `tari-university` will be created.

    3. Execute `cd tari-university` to go to that folder.

    4. Execute `git checkout -b da-regulatory-formatting-editing` to change to the working branch.

    4. Execute `git branch --set-upstream-to=origin/da-regulatory-formatting-editing da-regulatory-formatting-editing`  to track the working branch.

    5. Execute `git pull` to get the latest changes on that branch.

3. Run mdbook locally to render the working TLU website in the browser

    1. Make sure you are still in the main tari-university folder, it should be `~/tari-university`. If you execute `ls -l` you should see a file called `book.toml`, then you know you are in the correct place.

    2. Execute `mdbook serve`

    3. Open `http://localhost:3000` in a browser, then you should see your report in chapter 5.1

Regards

anselld commented:
Hi @lwillacy and @hansieodendaal , thanks very much for your comments. I've updated the report accordingly and will commit and push the changes. Please let me know if you have any further comments.
Regards, Denise

anselld commented:
Hi @lwillacy and @hansieodendaal 
The reference links that were not active have been fixed.

anselld commented:
Hi @hansieodendaal and @lwillacy
Hansie,  Louis is happy with the report. Please could you let us know what the next step in the process is. Thanks.

hansieodendaal commented:
Perfect @anselld and @lwillacy , will merge this to master.
The presentation can be done in a new PR on the same branch.

hansieodendaal commented:
OK, we can create a new branch for the presentation. I will add the basic files for it.
